extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -99,11 +105,20 @@ export class Bind {
     this.resources_ = resourcesForDoc(ampdoc);
 
     /**
+     * True if all bindings in the document have been scanned and parsed.
+     * @private {boolean}
+     */
+    this.initialized_ = false;
+
+    /**
      * True if a digest is triggered before scan for bindings completes.
      * @private {boolean}
      */
     this.digestQueuedAfterScan_ = false;
 
+    /** @const @private {boolean} */
+    this.workerExperimentEnabled_ = isExperimentOn(this.win_, 'web-worker');
```

Comment:
We also need to check that web workers are supported. Might be better to export a function from the `amp-worker.js` module.